crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -123,38 +241,108 @@ fn removed_argument(checker: &mut Checker, qualname: &QualifiedName, arguments:
                 None::<&str>,
             ));
         }
-        ["airflow", .., "operators", "trigger_dagrun", "TriggerDagRunOperator"] => {
-            checker.diagnostics.extend(diagnostic_for_argument(
-                arguments,
-                "execution_date",
-                Some("logical_date"),
-            ));
-        }
-        ["airflow", .., "operators", "datetime", "BranchDateTimeOperator"] => {
-            checker.diagnostics.extend(diagnostic_for_argument(
-                arguments,
-                "use_task_execution_day",
-                Some("use_task_logical_date"),
-            ));
+        _ if is_airflow_auth_manager(qualname.segments()) => {
```

Comment:
nit: same argument as mentioned above to combine the various catch-all into a single `if else` block

Reply:
Sure! Updated!